src/client/debugger/Common/Utils.ts

```diff
@@ -112,6 +112,9 @@ export function getPythonExecutable(pythonPath: string): string {
 }
 
 function isValidPythonPath(pythonPath): boolean {
+    if (fs.existsSync(pythonPath)) {
+        return true;
+    }
```

Comment:
The passed `pythonPath` is really a custom `launcher.sh` in this case, like:
```
#!/bin/bash
# do stuff
$somePythonVersion main.py
```
where `main.py` is:
```
if os.environ.get('VS_PY_DEBUGGER'):
        import visualstudio_py_launcher
        # HERE: setup `sys.argv` for the launcher (which can better be passed as arguments).
        visualstudio_py_launcher.launch()
    else:
        # run in non-debug mode
```
with debugger options:
```
{
  stopOnEntry: true,
  console: 'none',
  program: '/dev/null',
  debugOptions: ['WaitOnAbnormalExit', 'WaitOnNormalExit', 'RedirectOutput'],
  pythonPath: 'path/to/launcher.sh',
  env: {
    NUCLIDE_DEBUGGER: '1',
  },
}
```